resources/views/components/warning-misconfiguration.blade.php

```diff
@@ -0,0 +1,27 @@
+<div class="hidden" style="font-size: 24px; height: 100vh;">
+	<h1>If you can read me, it means that you misconfigured Lychee.</h1>
+	<p style="font-size: 20px;">Please check that:
+	<ul>
+		<li style="font-size: 20px; margin-bottom: 10px;">your
+			<pre style="font-size: 16px; display:inline-block; margin: 0;">APP_URL</pre> is properly set to the correct url.<br>
+			For example:
+			<pre style="font-size: 16px; display:inline-block; margin: 0;">APP_URL=https://lychee.example.com</pre>
+		</li>
+		<li style="font-size: 20px; margin-bottom: 10px;">if you are working behind a reverse proxy, that
+			<pre style="font-size: 16px; display:inline-block; margin: 0;">TRUSTED_PROXIES</pre> is set.<br>
+			For example:
+			<pre style="font-size: 16px; display:inline-block; margin: 0;">TRUSTED_PROXIES=*</pre>.
```

Comment:
Please don't recommend the wildcard.

Reply:
What would you suggest then ? :/

Reply:
Personally I probably wouldn't include an example :man_shrugging: 

I just don't like including hacky workarounds in official documentation.